backend/service-quiz/src/main/scala/pl/smtc/smartwords/ServiceQuizApp.scala

```diff
@@ -0,0 +1,41 @@
+package pl.smtc.smartwords
+
+import cats.effect._
+import com.comcast.ip4s._
+import org.http4s._
+import org.http4s.implicits._
+import org.http4s.server._
+import org.http4s.server.middleware._
+import org.http4s.ember.server._
+import pl.smtc.smartwords.controller._
+
+import scala.concurrent.duration.DurationInt
+
+object ServiceQuizApp extends IOApp {
+
+  override def run(args: List[String]): IO[ExitCode] = {
+    val healthController: HealthController = new HealthController()
+    val quizController: QuizController = new QuizController()
+
+    val config = CORSConfig(anyOrigin = true, allowCredentials = true, 1.day.toSeconds, anyMethod = true)
+    val apis = Router(
+      "/health" -> CORS(healthController.getRoutes, config),
+      "/quiz" -> CORS(quizController.getRoutes, config)
+    ).orNotFound
+    for {
+      server <- EmberServerBuilder.default[IO]
+        .withHost(ipv4"0.0.0.0")
+        .withPort(port"2222")
```

Comment:
We should add a configuration file with basic settings like port number, idle timeout, etc.

Reply:
This will be done in a separate change. Issue created.